### DISCUSSION TITLE
server port in build mode

### DISCUSSION THREAD

**Discussion by borodadada** [nuxt/nuxt#17441] [Questions] [answered]
Hi! I set port in dev mode use cross-env

package.json
```
"dev": "cross-env NUXT_PORT=3010 nuxt dev",
```

but how set port in build ? always run 3000

```
node .output/server/index.mjs 
Listening on http://0.0.0.0:3000/
```

**Comment by Diizzayy** [comment, marked as answer]
@borodadada use `--port` in your dev script or the `PORT` environment variable

- Dev Server

For the dev server you can simply set the `PORT` or `NUXT_PORT` environment variables

or use the `--port` flag as seen below

```json
{
  "scripts": {
    "dev": "nuxt dev --port 8001"
  }
}
```

- For the production server you can simply set the `PORT` or `NITRO_PORT` environment variables.

```sh
PORT=8001 node .output/server/index.mjs
```

**Reply by borodadada** [reply]
Okey done! thx
"start": "cross-env NITRO_PORT=3010 node .output/server/index.mjs"
